src/command_buffer.rs

```diff
@@ -120,6 +123,11 @@ impl CommandBuffer {
         self.remove::<(T,)>(ent);
     }
 
+    /// Updates the world with the given command after all other operations have run
+    pub fn command<F: Fn(&mut World) + Send + Sync + 'static>(&mut self, command_fn: F) {
```

Comment:
I wonder if this needs to be generic and pay for the associated code size overhead? Maybe this could take `Command` and that could have a

```rust
impl From<F> for Command where F: FnOnce(&mut World) + Send + Sync + 'static
```

to keep the generic as limited as possible? Or does this hurt the ergonomics of using it too much?

Reply:
As a one-liner, this being generic isn't going to impose a significant cost. A generic `From` impl would also have the same cost.